buildSrc/src/main/kotlin/Projects.kt

```diff
@@ -29,7 +29,17 @@ fun Project.setupAppModule(block: BaseAppModuleExtension.() -> Unit = {}) {
             versionName = project.versionName
             resConfigs("en")
             vectorDrawables.useSupportLibrary = true
+            multiDexEnabled = true
         }
+        buildTypes {
+            named("release") {
```

Comment:
I think this is `coil-sample`-specific configuration that we don't want to pull out into `setupAppModule` - same with `multiDexEnabled`.